src/pages/MemberProfileScreen.tsx

```diff
@@ -0,0 +1,21 @@
+import { Divider, Stack, Typography } from '@mui/material';
+
+import MemberPersonalInformation from '@/components/main/MemberPersonalInformation';
+import MemberPublicProfile from '@/components/main/MemberPublicProfile';
+import { useAccountTranslation } from '@/config/i18n';
+
+const MemberProfileScreen = (): JSX.Element | null => {
+  const { t } = useAccountTranslation();
+  return (
+    <Stack spacing={2}>
+      <Typography variant="h4" component="h1">
+        {t('PROFILE_TITLE')}
+      </Typography>
+      <Divider sx={{ my: 2 }} />
```

Comment:
The `my` has no effect on this component (at least from what I can see)
```suggestion
      <Divider />
```